### PR TITLE
Solve the four parts of the assignment (data inspection, standarizati…

### PR DESCRIPTION
**Pull request by jairo-aguirre**
…on, cross-validation, model accuracy)

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding new code to answer the four questions related to the assignment (data inspection, standardization, cross-validation, model accuracy)

## What did you learn from the changes you have made?
A lot!... I had to revisit Julia's lectures and researching a bit about the reasoning behind all the steps in the data processing

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
No really, I based my answers mostly on the lectures and added more from research for Q3

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
The challenges came from understanding the data preparation process and the key functions from **sklearn**. But I think I got it now.

## How were these changes tested?
By testing each line of the code and making sure I was getting the desired results

## A reference to a related issue in your repository (if applicable)
[scikit-learn](https://scikit-learn.org/stable/modules/neighbors.html)
[datacamp](https://www.datacamp.com/tutorial/k-nearest-neighbor-classification-scikit-learn)
[realpython](https://realpython.com/knn-python/#a-step-by-step-knn-from-scratch-in-python)

## Checklist
- [✅] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by efantinatti** [comment]
Hi Jairo

# PR Feedback Report

**Student**: Jairo  
**GitHub Username**: @jairo-aguirre 
**PR Number**: 1  
**Branch**: assignment-1  
**Status**: INCOMPLETE  
**PR URL**: https://github.com/jairo-aguirre/LCR/pull/1

## What You Did Well (Hits)

The pull request exists and is accessible. You correctly used the 'assignment-1' branch. Question 1: The student correctly identified the number of observations (178), variables (14), response variable type (int64), and levels ([0 1 2]). Question 2: You correctly applied StandardScaler and performed the train/test split with the correct proportions and random state. The explanations for standardization and random seed were accurate.

## Areas for Improvement (Errors)

Missing cell outputs.

## Overall Assessment

Resubmit it with the cell execution's outputs.

**Comment by efantinatti** [comment]
Hi @jairo-aguirre be sure you are using the scaled data as a origin for train and test.

**Comment by efantinatti** [comment]
Great job @jairo-aguirre. Completed.
